umm-spec-lib/resources/xml-schemas/echo10/Collection.xsd

```diff
@@ -260,7 +260,27 @@ xmlns:xs="http://www.w3.org/2001/XMLSchema">
             <xs:enumeration value="NEAR_REAL_TIME">
               <xs:annotation>
                 <xs:documentation> Data from the source that are
-                available for use within a time that is short in
+                available for use within a time that is short(1-3 hours) in
+                comparison to important time scales in the phenomena
+                being studied. This data is not science quality and
+                is not retained by EOSDIS once the SCIENCE_QUALITY
+                product is archived </xs:documentation>
+              </xs:annotation>
+            </xs:enumeration>
+            <xs:enumeration value="LOW_LATENCY">
+              <xs:annotation>
+                <xs:documentation> Data from the source that are
+                available for use within a time that is short(4-24 hours) in
+                comparison to important time scales in the phenomena
+                being studied. This data is not science quality and
+                is not retained by EOSDIS once the SCIENCE_QUALITY
+                product is archived </xs:documentation>
+              </xs:annotation>
+            </xs:enumeration>
+            <xs:enumeration value="EXPEDITED">
+              <xs:annotation>
+                <xs:documentation> Data from the source that are
```

Comment:
On the new line 282 the documentation for EXPEDITED can you replace the text with the following:
The data is ready for end user consumption 1 to 4 days after data acquisition. This data is not fully processed and is not retained by EOSDIS once the SCIENCE_QUALITY product is archived.